src/client/driver/command-executors/prepare-browser-manipulation.js

```diff
@@ -47,5 +49,14 @@ export default function prepareBrowserManipulation (browserId) {
 
             return delay(RESTORE_DOCUMENT_TITLE_TIMEOUT);
         })
-        .then(() => new DriverStatus({ isCommandResult: true, result }));
+        .then(() => {
+            if (result.isError) {
```

Comment:
Can't we have early error in `resize` action? We have max available dimensions on startup, can't we check against them before going to client?  Note that it should work in raw API as well